airflow/api_connexion/openapi/v1.yaml

```diff
@@ -1315,6 +1315,7 @@ components:
         extra:
           type: string
           readOnly: true
+          nullable: true
```

Comment:
Can you tell me more about it?

Reply:
From the model init method, it looks to me that the` extra` parameter can be nullable.
https://github.com/apache/airflow/blob/dcf65765e58fb8beba206454075bbd6675b65721/airflow/models/log.py#L46-L67

Reply:
And if it is None then on serializing it, it would come out as None instead of a string. This would not conform to the spec

Reply:
I agree with it. Could you do it in a separate PR? I would like to keep the specification change history clean. There is a good chance that someone will want to do git blame on the specification.

Reply:
Making the PR now, will be through in a few minutes

Reply:
Here is the PR https://github.com/apache/airflow/pull/9315